web/src/pages/blog/index.astro

```diff
@@ -0,0 +1,15 @@
+---
+import MainLayout from "../../layouts/global.astro"
+import PostCard from "../../components/post.astro"
+
+const allPosts = await Astro.glob("./*.mdx")
+---
+
+<MainLayout title="Blog">
+	<section class="container" aria-label="New Blog Posts">
+		<h1 class="h1">New Blog Posts</h1>
```

Comment:
lul remove the `class=h1`. It doesn't do anything and is exceptionally useless anyway, since you can style h1 elements themselves without a class.

If it's not a tailwind class, don't add it.

Reply:
okay, making more sense. The classes before were used to identify the elements back in a CSS file. But now since we have tailwind, there should only be a class if there is styling happening in it.